PowerPointLabs/PowerPointLabs/ELearningLab/Converters/ShapeNameWidthConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Text.RegularExpressions;
```

Comment:
Unused import directives should be removed.

Reply:
Removed unused directive